### PR TITLE
Translations update from Hosted Weblate

### PR DESCRIPTION
**Pull request by weblate**
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Keycloak/Admin UI](https://hosted.weblate.org/projects/keycloak/admin-ui/).

It also includes following components:

* [Keycloak/Theme base/account](https://hosted.weblate.org/projects/keycloak/theme-baseaccount/)

* [Keycloak/Theme base/admin](https://hosted.weblate.org/projects/keycloak/theme-baseadmin/)

* [Keycloak/Theme base/email](https://hosted.weblate.org/projects/keycloak/theme-baseemail/)

* [Keycloak/Theme base/login](https://hosted.weblate.org/projects/keycloak/theme-baselogin/)

* [Keycloak/Account UI](https://hosted.weblate.org/projects/keycloak/account-ui/)

Current translation status:

Weblate translation status

### CODE REVIEW

File: themes/src/main/resources-community/theme/base/account/messages/messages_de.properties [hunk 1]
```diff
@@ -339,3 +339,46 @@ access-denied-when-idp-auth=Zugriff verweigert bei Authentifizierung mit {0}
...
 invalidPasswordNotContainsUsernameMessage=Ungültiges Passwort: Darf den Benutzernamen nicht enthalten.
+locale_ar=عربي
+locale_ja=日本語
+locale_it=Italiano
+error-user-attribute-required=Bitte geben Sie Attribut {0} an.
+locale_ru=Русский
+locale_uk=Українська
+locale_zh-CN=中文简体
+locale_tr=Türkçe
+locale_fi=Suomi
```
Comment (jonkoops) [inline themes/src/main/resources-community/theme/base/account/messages/messages_de.properties:350]: Hmmm, I thought we were going to get rid of this, should we hold off on this? @ahus1

File: themes/src/main/resources-community/theme/base/account/messages/messages_de.properties [hunk 2]
```diff
@@ -339,3 +338,21 @@ access-denied-when-idp-auth=Zugriff verweigert bei Authentifizierung mit {0}
...
+error-invalid-value=Ungültiger Wert.
+error-empty=Bitte geben Sie einen Wert an.
+error-invalid-length=Das Attribut {0} muss eine Länge zwischen {1} und {2} haben.
+error-invalid-length-too-short=Das Attribut {0} muss mindestens die Länge {1} haben.
+error-number-out-of-range-too-small=Das Attribut {0} muss mindestens den Wert {1} haben.
+error-number-out-of-range-too-big=Das Attribut {0} darf maximal den Wert {2} haben.
+error-pattern-no-match=Ungültiger Wert.
+error-invalid-uri-scheme=Ungültiges URL-Schema.
+error-invalid-date=Ungültiges Datum.
+error-user-attribute-read-only=Das Feld {0} ist read only.
```
Comment (ahus1) [inline themes/src/main/resources-community/theme/base/account/messages/messages_de.properties:351]: Could you please try to find a translation for "read only" as well?

**Review by ahus1 — CHANGES_REQUESTED** [review]
Thank you @robson90 for the translations. See one comment below where it needs IMHO a refinement.

File: themes/src/main/resources-community/theme/base/account/messages/messages_de.properties (hunk 1 above)
Comment (ahus1) [reply themes/src/main/resources-community/theme/base/account/messages/messages_de.properties]: The locales appeared in to message resources, and I set them to empty and not translated only in one of them. I fixed it now in the Weblate UI.

File: themes/src/main/resources-community/theme/base/account/messages/messages_de.properties (hunk 2 above)
Comment (robson90) [reply themes/src/main/resources-community/theme/base/account/messages/messages_de.properties:351]: Then "schreibgeschützt"? Or something like "Auf das Feld {0} haben Sie nur lesenden Zugriff"

Comment (ahus1) [reply themes/src/main/resources-community/theme/base/account/messages/messages_de.properties:351]: I'd think I'd go for "schreibgeschützt".

Comment (robson90) [reply themes/src/main/resources-community/theme/base/account/messages/messages_de.properties]: Fixed.

**Review by ahus1 — DISMISSED** [review]
Thank you for these translations!

**Review by ahus1 — APPROVED** [review]

**Review by jonkoops — APPROVED** [review]

**Pull request merged**